src/BL/Services/IMinioHelper.cs

```diff
@@ -28,5 +29,8 @@ public interface IMinioHelper
         Task<bool> BucketPolicySetPublic(string bucketName);
 
         Task DeleteObject(string bucketName, string objectKey);
+        Task<MinIOOperationResult> CreateUserWithMcAsync(string accessKey, string secretKey);
```

Comment:
```suggestion
        Task<MinIOOperationResult> CreateUserAsync(string accessKey, string secretKey);
```

WithMc is an implementation detail.

Arguably put that it's used in a doc comment to inform people how it works, but that should go on the implementation not the interface.